modules/network-observability-packet-drops.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-packet-drops_{context}"]
+= Working with packet drops
+Packet loss occurs when one or more packets of network flow data fail to reach their destination. You can track these drops by enabling the `spec.agent.ebpf.features` and `spec.agent.ebpf.privileged` specifications in the `FlowCollector`.
++
+[IMPORTANT]
+====
+Cpu and memory usage increases when this feature is enabled.
+====
+
+.Prerequisites
+* Access to an {product-title} cluster with version 4.13. 
+* Kernel supported by Red Hat Enterprise Linux (RHEL) 9.2.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-pkt-drop_{context}"]
+.Configure `FlowCollector` for packet drops
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features: PKT_DROP            <1>                         
+      privileged: true              <2>
+----
+<1> You can set the `spec.agent.ebpf.feature` to `PKT_DROP` enable reporting of packet drops of each network flow in the web console. 
```

Comment:
similarly has to reflect that its a list

Reply:
Ok I addressed here and will also address in the DNS PR. TY for mentioning this!